api-reference/echo-operations/README.md

```diff
@@ -10,25 +10,25 @@
 
 ## Asset Management
 
-* [asset_create_operation](asset_management.md#asset_create_operation) - Create new asset
-* [asset_update_operation](asset_management.md#asset_update_operation) - Update options common to all assets
-* [asset_update_bitasset_operation](asset_management.md#asset_update_bitasset_operation) - Update options specific to BitAssets
-* [asset_update_bitasset_operation](asset_management.md#asset_update_bitasset_operation) - Update options specific to BitAssets
-* [asset_update_feed_producers_operation](asset_management.md#asset_update_feed_producers_operation) - Update the set of feed-producing accounts for a BitAsset
-* [asset_issue_operation](asset_management.md#asset_issue_operation)
-* [asset_reserve_operation](asset_management.md#asset_reserve_operation) - Used to take an asset out of circulation, returning to the issuer
-* [asset_fund_fee_pool_operation](asset_management.md#asset_fund_fee_pool_operation)
-* [asset_publish_feed_operation](asset_management.md#asset_publish_feed_operation) - Publish price feeds for market-issued assets
+* [asset_create_operation](asset-management.md#asset_create_operation) - Create new asset
+* [asset_update_operation](asset-management.md#asset_update_operation) - Update options common to all assets
+* [asset_update_bitasset_operation](asset-management.md#asset_update_bitasset_operation) - Update options specific to BitAssets
+* [asset_update_bitasset_operation](asset-management.md#asset_update_bitasset_operation) - Update options specific to BitAssets
+* [asset_update_feed_producers_operation](asset-management.md#asset_update_feed_producers_operation) - Update the set of feed-producing accounts for a BitAsset
+* [asset_issue_operation](asset-management.md#asset_issue_operation)
```

Comment:
Add a brief description of the "asset_issue_operation" operation.

Reply:
Fixed